### ISSUE TITLE
Vertical line on every graph

### ISSUE THREAD

**Issue by HellSpawn754** [munin-monitoring/munin#1638]
Hello!

Thanks for the great software, but i have a small problem:

![374994751-a1418937-4cd3-4510-bfa6-9d0a27f5411c](https://github.com/user-attachments/assets/bd2100d2-9478-4957-92fc-e5a7a92323ad)

I have a black vertical line at every graph that in other munin setups appear when no data is available.

But all graphs update normal. (disregard the missing data. it was me testing. atm data is beeing fetched normaly)
/var/log/munin/munin-update.log has no errors.
For example "2024/10/09 16:10:07 [INFO]: Munin-update finished (7.43 sec)"

Is there anything i can do to remove this line? Or to check why it does appear?

Thank you!

**Comment by steveschnepp** [comment]
It is the bar of last update IIRC

**Comment by HellSpawn754** [comment]
Yes :-) but when the last update was ok this line should not be rendered. In my case it is always there. 

In "/var/log/munin/munin-update.log" i have no errors.

**Comment by steveschnepp** [comment]
Well, it is currently rendered unconditionally. And the last run might not be in error, just late.
